src/main/java/org/myrobotlab/service/interfaces/SearchPublisher.java

```diff
@@ -13,7 +14,7 @@ public interface SearchPublisher extends TextPublisher, LocaleProvider {
 
   SearchResults publishResults(SearchResults results);
```

Comment:
please specify which methods are published.. and add the default implementation of the attach/remove SearchListener

Reply:
search replace error corrected